fhir-term/src/main/java/com/ibm/fhir/term/exception/FHIRTermException.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.term.exception;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+import com.ibm.fhir.model.resource.OperationOutcome.Issue;
+
+public class FHIRTermException extends RuntimeException {
```

Comment:
I assume this is a user error

Reply:
This exception is generated by utility classes local to the fhir-term packages whereas `FHIRTermServiceException` is intended to come through `FHIRTermService` and `FHIRTermServiceProvider` implementations.

Reply:
Will add Javadoc